corgi/tasks/management/commands/loadunprocessedrelations.py

```diff
@@ -14,15 +14,8 @@ def add_arguments(self, parser: CommandParser) -> None:
             dest="product_ref",
             help="Fetch unprocessed relations for product reference",
         )
-        parser.add_argument(
-            "-s",
-            "--save_only",
-            action="store_true",
-            help="Call fetch anyway to save the product taxonomy if build already exists",
-        )
 
     def handle(self, *args, **options):
-        processed_builds = 0
         if options["product_ref"]:
             if (
                 not ProductComponentRelation.objects.db_manager("read_only")
```

Comment:
Not related to this change, and I guess it's been working fine, but I'd be worried about using the read-only DB for tasks which need to process data / write results back. If there were a large amount of replication lag, we could get stale data from the read-only copy and write it back to the default DB, which could cause weird heisenbugs that are very hard to troubleshoot.

In this case, I think we'd fail to process some of the unprocessed relations, whenever they've been created in the default DB but haven't yet been replicated to the read-only copy.
```suggestion
                not ProductComponentRelation.objects
```

Reply:
It's possible I suppose, removing read_only here.